### PR TITLE
Add tools to tools.json and fix surelog SC-169

### PR DESCRIPTION
**Pull request by gadfort**

### CODE REVIEW

**Comment by aolofsson** [comment]
Noticed the dependence on "flopoco" in the panda build. Can you remove it, it's not a proper open source project.

**Comment by gadfort** [comment]
> Noticed the dependence on "flopoco" in the panda build. Can you remove it, it's not a proper open source project.

Remove the option flopoco in the build or remove the panda build all together?

**Comment by aolofsson** [comment]
Just the flopoco. Panda should be fine, although I'm am not sure we have
reviewed the licenses for all the install tools..?

File: setup/install-klayout.sh [hunk 1]
```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 
+# Get directory of script
+src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
+
 mkdir -p deps
 cd deps
 
-pkg_version="0.27.8-1"
+pkg_version=$(python3 ${src_path}/_tools.py --tool klayout --field version)
```
Comment (WRansohoff) [inline setup/install-klayout.sh:9]: My gut says that sed or awk might be better for extracting strings in a bash script, but JSON isn't fun to parse that way, and this *is* a Python project. Who knows, maybe someday we'll move these setup scripts from .sh to .py.

**Review by WRansohoff — APPROVED** [review]

File: setup/install-klayout.sh (hunk 1 above)
Comment (gadfort) [reply setup/install-klayout.sh:9]: sed or awk would be pretty hard to manage. The point of the _tools.json file is to have the versions in a single place and the python script here helps get the information out. We could look at making python scripts, but given how finicky these install scripts are to begin with I don't know if we want to be the single source of install scripts.

**Pull request merged**